src/transformState.js

```diff
@@ -62,7 +62,27 @@
  * @param {Object[]} transforms
  */
 function transformState(state, transforms) {
-  // write code here
+  for (const key of transforms) {
+    for (const value in key) {
```

Comment:
What do key and value mean here? Transformation and command would be better names for these variables